src/main/kotlin/no/nav/amt/distribusjon/varsel/VarselService.kt

```diff
@@ -64,61 +66,84 @@ class VarselService(
     private fun opprettBeskjed(hendelse: Hendelse) = opprettVarsel(
         hendelse = hendelse,
         type = Varsel.Type.BESKJED,
-        aktivTil = nowUTC().plus(beskjedAktivLengde),
-        tekst = PLACEHOLDER_BESKJED_TEKST,
     )
 
     private fun opprettPameldingsoppgave(hendelse: Hendelse) = opprettVarsel(
         hendelse = hendelse,
         type = Varsel.Type.OPPGAVE,
-        aktivTil = null,
-        tekst = PAMELDING_TEKST,
     )
 
-    private fun opprettVarsel(
-        hendelse: Hendelse,
-        type: Varsel.Type,
-        aktivTil: ZonedDateTime?,
-        tekst: String,
-    ) {
-        repository.getByHendelseId(hendelse.id).onSuccess {
-            log.info("Varsel for hendelse ${hendelse.id} er allerede opprettet. Oppretter ikke nytt varsel.")
-            return
-        }
-
-        val forrigeVarsel = repository.getSisteVarsel(hendelse.deltaker.id, type).getOrNull()
+    private fun sendVarsel(varsel: Varsel) {
+        val forrigeVarsel = repository.getSisteVarsel(varsel.deltakerId, varsel.type).getOrNull()
         if (forrigeVarsel?.erAktiv == true) {
```

Comment:
Her bør det nok være noe mer logikk for å si om vi skal varsle på nytt eller ikke. F.eks så bør nok beskjeder fornyes, slik at de blir visendes i 3 uker fra siste sendte varsel.